packages/iov-bns/src/client.spec.ts

```diff
@@ -52,12 +52,12 @@ describe("Integration tests with bov+tendermint", () => {
   // max open connections??? (but 900 by default)
   const tendermintUrl = "ws://localhost:22345";
 
-  const userProfile = async (): Promise<UserProfile> => {
+  async function userProfileWithFaucet(): Promise<UserProfile> {
```

Comment:
you really don't like my `async () =>` abuse do you?

Reply:
better name in any case

Reply:
I am trying hard to get used to it but it is still taking me considerably longer to parse that code when reading

Reply:
The only time I think it really shines are either inline functions or partial ones...

```
const parse = (config: Config) => (item: Item) => {
  // do somethng with item based on config values
}
const parsed = items.map(parse(myConfig));
```

type def on top line reads almost like Haskell ;)